quickstatements/latest/Dockerfile

```
@@ -5,7 +5,12 @@ RUN apt-get update && \
     apt-get clean && rm -rf /var/lib/apt/lists/*
 
 RUN git clone https://phabricator.wikimedia.org/source/tool-quickstatements.git quickstatements
-RUN git clone --depth 1 https://bitbucket.org/magnusmanske/magnustools.git magnustools
+
+RUN git clone https://bitbucket.org/magnusmanske/magnustools.git magnustools && \
+    git --git-dir=./magnustools/.git checkout d791f84cba556a29fa5c84543b3c4717a0c63876
```

This doesn't work as you change only the git dir but don't tell git which working dir to use.

```suggestion
    git -C magnustools checkout d791f84cba556a29fa5c84543b3c4717a0c63876
```

To quote the last line of the documentation on [--git-dir](https://git-scm.com/docs/git#Documentation/git.txt---git-dirltpathgt):

> If you just want to run git as if it was started in `<path>` then use `git -C <path>`.


Unless you were intending to do something else?